server/src/routes/auth.ts

```diff
@@ -142,3 +143,21 @@ auth.get("/verify-token", jwt, async (req, res, next) => {
 		next(e)
 	}
 })
+
+auth.get("/signout", jwt, async (req, res, next) => {
+	try {
+		const authHeader = req.headers["authorization"]
+		const token = authHeader && authHeader.split(" ")[1]
+		const blacklist = await new JWTDenyList({token:token})
+		await blacklist.save()
+		req.headers["authorization"] = ''
+		res.status(200).json({
+			message: "You are now logged out",
+			token: `Expired token ${token}`
```

Comment:
Just send token, no need for the “Expired token “ part of the string. Or, just send “expiredToken” and you can remove the message field too